dev/integration_tests/flutter_gallery/macos/Podfile.lock

```diff
@@ -1,35 +1,29 @@
 PODS:
-  - connectivity_macos (0.0.1):
-    - FlutterMacOS
-    - Reachability
   - FlutterMacOS (1.0.0)
-  - Reachability (3.2)
   - url_launcher_macos (0.0.1):
     - FlutterMacOS
+  - video_player_avfoundation (0.0.1):
```

Comment:
Random lock file change 🙂

Reply:
Guessing this was introduced by the test run? Meant not to commit this but pretty sure I `git commit -a`'ed it mindlessly. Should this file be gitignored?